File: SoloLearn/Easter Eggs/EasterEggs.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int eggs = Convert.ToInt32(Console.ReadLine()), eggsBasket = Convert.ToInt32(Console.ReadLine()), eggsFriend = Convert.ToInt32(Console.ReadLine());
            if (eggs = eggsBasket + eggsFriend)
                Console.WriteLine("Candy Time");
            else 
                Console.WriteLine("Keep Hunting");
        }
    }
} 

File: Code Coach/Easter Eggs/EasterEggs.swift
if let eggs = readLine(), let eggsBasket = readLine(), let eggsFriend = readLine(), let e = Int(eggs), let b = Int(eggsBasket), let f = Int(eggsFriend) {
    if e == b + f {
        print("Candy Time")
    } else {
        print("Keep Hunting")
    }
}